WMF Framework/WMFExploreFeedContentController.m

```diff
@@ -75,23 +75,24 @@ - (WMFNearbyContentSource *)nearbyContentSource {
 
 - (NSArray<id<WMFContentSource>> *)contentSources {
     NSParameterAssert(self.dataStore);
-    NSParameterAssert([self siteURL]);
+    NSParameterAssert(self.siteURLs);
     if (!_contentSources) {
-        WMFFeedContentSource *feedContentSource = [[WMFFeedContentSource alloc] initWithSiteURL:[self siteURL]
-                                                                                  userDataStore:self.dataStore
-                                                                        notificationsController:[WMFNotificationsController sharedNotificationsController]];
-        feedContentSource.notificationSchedulingEnabled = YES;
-        _contentSources = @[
-            [[WMFRelatedPagesContentSource alloc] init],
-            [[WMFMainPageContentSource alloc] initWithSiteURL:[self siteURL]],
-            [[WMFContinueReadingContentSource alloc] initWithUserDataStore:self.dataStore],
-            [[WMFNearbyContentSource alloc] initWithSiteURL:[self siteURL]
-                                                  dataStore:self.dataStore],
-            feedContentSource,
-            [[WMFRandomContentSource alloc] initWithSiteURL:[self siteURL]],
-            [[WMFAnnouncementsContentSource alloc] initWithSiteURL:[self siteURL]],
-            [[WMFOnThisDayContentSource alloc] initWithSiteURL:[self siteURL]]
-        ];
+        NSMutableArray *mutableContentSources = [NSMutableArray arrayWithCapacity:2 + self.siteURLs.count * 7];
```

Comment:
Wondering if we'll forget to adjust `7` here next time we add/remove a content source type...

Reply:
There's not a huge penalty for the capacity being off in this case since this isn't performance critical or called frequently, we probably don't even need to set it here. Moreso just habit at this point